defaults/defaults.env

```diff
@@ -21,9 +21,11 @@ export ELECTRA_FORK_VERSION="${ELECTRA_FORK_VERSION:-0x60000000}"
 export ELECTRA_FORK_EPOCH="${ELECTRA_FORK_EPOCH:-2000}"
 export FULU_FORK_VERSION="${FULU_FORK_VERSION:-0x70000000}"
 export FULU_FORK_EPOCH="${FULU_FORK_EPOCH:-99999}"
-export EIP7732_FORK_VERSION="${EIP7732_FORK_VERSION:-0x80000000}"
+export EIP7441_FORK_VERSION="${EIP7441_FORK_VERSION:-0x80000000}"
+export EIP7441_FORK_EPOCH="${EIP7441_FORK_EPOCH:-99999}"
+export EIP7732_FORK_VERSION="${EIP7732_FORK_VERSION:-0x90000000}"
 export EIP7732_FORK_EPOCH="${EIP7732_FORK_EPOCH:-99999}"
-export EIP7805_FORK_VERSION="${EIP7805_FORK_VERSION:-0x90000000}"
+export EIP7805_FORK_VERSION="${EIP7805_FORK_VERSION:-0x10000000}"
```

Comment:
why is a fork at 0x100.. as the default? we go up in numbers, 1xxx is for genesis

Reply:
9+1 = 10 lol

Reply:
should probably use something else